hw/ip/aes/pre_syn/syn_setup.example.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Setup IP name and top module.
+# When changing the top module, some parameters might not exist and
+# tcl/yosys_run_synth.tcl might need to be adjusted accordingly.
+export LR_SYNTH_IP_NAME=aes
+export LR_SYNTH_TOP_MODULE=aes
+
+# Setup module parameters.
+export LR_SYNTH_AES_192_ENABLE=1
+export LR_SYNTH_MASKING=1
+export LR_SYNTH_S_BOX_IMPL=4
+
+# Setup cell library path.
+# Uncomment the lines below and set the path to an appropriate .lib file
+# export LR_SYNTH_CELL_LIBRARY_PATH=/path/to/NangateOpenCellLibrary_typical.lib
+# export LR_SYNTH_CELL_LIBRARY_NAME=nangate
```

Comment:
For extra friendliness, it might be good to follow this block with a check like `if [ -z "$LR_SYNTH_CELL_LIBRARY_PATH" ]; echo >&2 'You forgot to set LR_SYNTH_CELL_LIBRARY_PATH!'; exit 1; fi` and similar for the name?

Reply:
Good idea! I've added this for both these variables.